Lib/test/test_c_locale_coercion.py

```diff
@@ -405,7 +405,11 @@ def test_LC_ALL_set_to_C(self):
 
     def test_PYTHONCOERCECLOCALE_set_to_one(self):
         # skip the test if the LC_CTYPE locale is C or coerced
-        old_loc = locale.setlocale(locale.LC_CTYPE, None)
+        try:
+            old_loc = locale.setlocale(locale.LC_CTYPE, None)
```

Comment:
This call works, it just gets the current locale.

locale.setlocale(locale.LC_CTYPE, "") is the one which fails: it sets LC_CTYPE to the user preferred locale.